app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>oompr</h1>
+
+<h3>Where do you want to live?</h3>
+
+<%= form_tag(search_path, method: :get) do %>
+  <%= text_field_tag :city, params[:city] %>
```

Comment:
should we have a place holder here ? Just so it is not an empty block.

Reply:
I think it makes sense for the search field to be empty - I haven't seen many websites that pre-populate a search field for you